src/js/Hls.js

```diff
@@ -7,6 +7,7 @@ class Hls extends Meister.MediaPlugin {
     constructor(config, meister, next) {
         super(config, meister);
 
+        this.maxErrors = this.config.maxErrors || 5;
```

Comment:
Perhaps make the config something like stallingThreshold, as this is really what we're trying to achieve.